server/templates/mainPages/emissions.html

```diff
@@ -78,4 +79,10 @@ <h3>All Sector GHG Data Chart</h3>
 </div>
 {% endif %}
 
+<!--TODO: Update link to actual recommendations page-->
+<br>
+<div class="container">
+  <button onclick="window.location='http://127.0.0.1:5000/recommendations'">Recommendations</button>
```

Comment:
A couple things to note here: First, it seems iframes dont work when the app is deployed to the server.  A normal link would be fine.  Second, this link is actually linking to a local host instead of the host on the server that is hosting the site (tinyfeet.app).  Really, the ideal way to link to another page using flask jinja templates is with {{ url_for('recommendations') }}.